### DISCUSSION TITLE
Recreate docker container

### DISCUSSION THREAD

**Discussion by Pingear** [serverpod/serverpod#677] [Q&A] [answered]
Hi) I accidentally deleted the project's docker container, is there any way to generate a container for an existing project?

**Comment by yahu1031** [comment]
in what os(mac/windows/linux)?

**Reply by Pingear** [reply]
mac)

**Reply by yahu1031** [reply]
Windows - Checkout [setup-tables.cmd](https://github.com/serverpod/serverpod/blob/main/templates/serverpod_templates/projectname_server/setup-tables.cmd#L5) - edited

Mac/Linux - Checkout [setup-tables](https://github.com/serverpod/serverpod/blob/main/templates/serverpod_templates/projectname_server/setup-tables#L5) - edited

**Reply by yahu1031** [reply]
a simple command, but later you need to do other things. If you ask me, copy this and change the data and run it.

**Reply by Pingear** [reply]
Thanks It works)

**Reply by Pingear** [reply]
Sorry, I was in a hurry, it still doesn’t work, but the problem is not that I deleted the container, but in a completely new project I have this error - 
SERVERPOD version: 1.0.1 mode: development time: 2023-02-18 20:38:57.072212Z
2023-02-18 20:38:57.456836Z Failed to connect to database.
PostgreSQLSeverity.fatal 28P01: password authentication failed for user "postgres" 
===== asynchronous gap ===========================
package:postgres_pool/postgres_pool.dart 351:18                 PgPool.run.<fn>
package:retry/retry.dart 131:16                                 RetryOptions.retry
package:postgres_pool/postgres_pool.dart 349:14                 PgPool.run
package:serverpod/src/database/database_connection.dart 171:20  DatabaseConnection.find
package:serverpod/src/database/database_connection.dart 199:18  DatabaseConnection.findSingleRow
package:serverpod/src/database/database.dart 86:12              Database.findSingleRow
package:serverpod/src/server/serverpod.dart 310:13              Serverpod.start.<fn>
package:serverpod/src/server/serverpod.dart 299:5               Serverpod.start
package:some_server/server.dart 33:3                            run

2023-02-18 20:38:58.804671Z Internal server error. Failed to load method lookup.
PostgreSQLSeverity.fatal 28P01: password authentication failed for user "postgres" 
===== asynchronous gap ===========================
package:postgres_pool/postgres_pool.dart 351:18                 PgPool.run.<fn>
package:retry/retry.dart 131:16                                 RetryOptions.retry
package:postgres_pool/postgres_pool.dart 349:14                 PgPool.run
package:serverpod/src/database/database_connection.dart 171:20  DatabaseConnection.find
package:serverpod/src/database/database_connection.dart 199:18  DatabaseConnection.findSingleRow
package:serverpod/src/database/database.dart 86:12              Database.findSingleRow
package:serverpod/src/server/method_lookup.dart 47:26           MethodLookup._attemptLoad
package:serverpod/src/server/method_lookup.dart 30:7            MethodLookup.load
package:serverpod/src/server/serverpod.dart 325:9               Serverpod.start.<fn>
package:serverpod/src/server/serverpod.dart 299:5               Serverpod.start
package:some_server/server.dart 33:3                            run

Insights listening on port 8081
Server default listening on port 8080
Webserver listening on port 8082
2023-02-18 20:38:59.046718Z Internal server error. Failed to connect to database in future call manager.
PostgreSQLSeverity.fatal 28P01: password authentication failed for user "postgres" 
===== asynchronous gap ===========================
package:postgres_pool/postgres_pool.dart 351:18                 PgPool.run.<fn>
package:retry/retry.dart 131:16                                 RetryOptions.retry
package:postgres_pool/postgres_pool.dart 349:14                 PgPool.run
package:serverpod/src/database/database_connection.dart 443:20  DatabaseConnection.deleteAndReturn
package:serverpod/src/database/database.dart 166:12             Database.deleteAndReturn
package:serverpod/src/server/future_call_manager.dart 100:18    FutureCallManager._checkQueue

Local stacktrace:
#0      FutureCallManager._checkQueue (package:serverpod/src/server/future_call_manager.dart:136:36)
<asynchronous suspension>

**Comment by Pingear** [comment, marked as answer]
The problem was that I had previously created a project with the **same name**. I'm not familiar with docker, but until you r**emove the corresponding volume from volumes**, the re-created project with the same name will not be able to connect to the database

**Reply by vlidholt** [reply]
Happy you figured it out. Thanks for posting your solution! :)

**Reply by yahu1031** [reply]
There was already a discussion on this. If there are any unused containers(irrespective of name) better to be cleared off before you recreate/create a project. So you can avoid these kinds of issues. In case you find it, it is not late after creating a new project too.
